packages/entity-server/src/routes/hierarchy/relationships/EntityRelationshipsRoute.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
+ */
+
+import { Route } from '@tupaia/server-boilerplate';
+import { RelationshipsRequest } from './types';
+import { ResponseBuilder } from './ResponseBuilder';
```

Comment:
Decided to just go with `ReponseBuilder` here over `RelationshipsResponseBuilder`. Kind of personal preference a bit, but when some functionality is private to a package, I like to omit the prefixes that are already in the package name. I know this can make code a little less searchable, and perhaps read a little unexpectedly at times, but I think I prefer that over ever increasingly verbose names... Happy to go either way at the end of the day 🤷

Reply:
I lean a little towards verbosity because I use `cmd` + `p` in vscode and it's more helpful for that form of navigation (always hate getting to the specific `index.js` file I need to edit...), but I also don't mind it and it doesn't take too much longer to see which folder the file is within :+1: